go/docs/_build/dirhtml/404.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
```

Comment:
@lauranovich Create a  ``.gitignore`` file in the root dir and add ``go/docs/_build/``. Then,  remove the ``go/docs/_build/``.

Reply:
@dgarcia360 - there already is a .gitignore file in the Go directory

1 - Do I add the go/docs/_build/ to it OR
2 - Do Iadd a new .gitignore file at the root level with just go/docs/_build in it or with all of the content from the one in the Go Directory?
AND
3 - Do I use the contents of the .gitignore file from the Theme directory or just a blank file with only go/docs/_build in it?

Reply:
If there is already a ``.gitignore`` file inside the ``go`` dir, list ``docs/_build`` in this file - there is no need to create a new one. 

Then, remove the generated ``go/docs/_build`` folder from git.

```
rm -r go/docs/_build
git add go/docs
git commit -m "Remove _build from git"
git push
````

Reply:
great! thanks

Reply:
I would advise doing

Reply:
I would suggest doing
```
rm -r go/docs/_build
git add go/docs
git commit --amend
git push -f
```
to not to keep code addition and removal in the branch history

Reply:
oh, I see the history. It would not work. But I can squash the PR then) or you can do history edit with
```
$ git checkout add-sphinx-theme
$ git commit -i --autosquash master
and kicking off addtions and removals
```

Reply:
Basically, squash will do

Reply:
so we're good?